azure-devops/projects/pagopa-packages-projects/express-azure-functions.tf

```diff
@@ -0,0 +1,96 @@
+variable "express-azure-functions" {
```

Comment:
```suggestion
variable "io-functions-express" {
```
change also refs

Reply:
Repository is called `io-functions-express`, but the package name is now `@pagopa/azure-functions-express`.
I thought to use the same name, where we have this opportunity, leaving the old one for  repository name only.
Can we do it? 
cc: @balanza

Reply:
variables and names on gitops refer to github repository.
We can set only repository.name = "io-functions-express" but maybe it's better to rename https://github.com/pagopa/io-functions-express

Reply:
I did your first option, but the second one is the best one, I agree. I don't think I can do it.

Reply:
Help @balanza :) Can u rename `io-functions-express` in `azure-functions-express`?

Reply:
done